### PR TITLE
I18n es

### PR DESCRIPTION
**Pull request by Gnefil**
i18n: src/components Spanish translation completed

### CODE REVIEW

File: src/components/subjects.vue [hunk 1]
```diff
@@ -678,6 +678,37 @@ export default {
...
+        "subject_name_hint": "E.g. Mathematical Techniques",
+        "short_name": "Nombre corto *",
+        "short_name_hint": "E.g. Math",
+        "subject_number": "Número de asginatura *",
+        "subject_number_hint": "E.g. COMP11120",
+        "subject_home": "Página principal de la asignatura *",
+        "subject_color": "Color de asignatura",
+        "hide_subject": "Esconder esta asignatura",
+        "add_link": "Añadir enlace de una sesión online",
+        "link_format": "URL[nombre][contraseña]",
```
Comment (yrccondor) [inline src/components/subjects.vue:703]: missing spaces after every left brackets

File: src/components/clock.vue [hunk 2]
```diff
@@ -1580,6 +1580,569 @@ p {
...
+        "Wallis and Futuna": "Wallis and Futuna",
+        "Chatham": "Chatham",
+        "Apia": "Apia",
+        "Samoa": "Samoa",
+        "Enderbury": "Enderbury",
+        "Fakaofo": "Fakaofo",
+        "Tokelau": "Tokelau",
+        "Nuku‘alofa": "Nuku‘alofa",
+        "Tonga": "Tonga",
+        "Kiritimati": "Kiritimati"
```
Comment (yrccondor) [inline src/components/clock.vue:2145]: Could you please remove all country/city names if they are as same as English?

**Review by yrccondor — CHANGES_REQUESTED** [review]

**Comment by Gnefil** [comment]
The requested changes are done, although countries and cities may take a while, we can place them in another pull request.

**Comment by Gnefil** [comment]
Changed self study in Spanish into shorter word

File: src/components/clock.vue [hunk 3]
```diff
@@ -1580,6 +1580,198 @@ p {
...
+        "Dededo Village": "Dededo",
+        "Northern Mariana Islands": "Islas Marianas del Norte",
+        "Port Moresby": "Puerto Moresby",
+        "Papua New Guinea": "Papúa Nueva Guinea",
+        "New Caledonia": "Nueva Caledonia",
+        "Norfolk Island": "Isla Norfolk",
+        "Solomon Islands": "Islas Salomón",
+        "Marshall Islands": "Islas Marshall",
+        "New Zealand": "Nueva Zelanda",
+        "Wallis and Futuna": "Wallis y Futuna",
```
Comment (yrccondor) [inline src/components/clock.vue:1774]: the extra ending comma leads to build failure

File: src/components/calendar.vue [hunk 4]
```diff
@@ -902,6 +902,23 @@ export default {
...
+        "to": "Hasta: ",
+        "course_ddl": "Fecha límite para trabajo de curso",
+        "coursework": "Trabajo de curso",
+        "subject_home": "Página principal de asignatura",
+        "ical_error": "No ha sido posible analizar los archivos ical.",
+        "network_error_body": "No ha sido posible obtener los últimos eventos desde la subscripción del calendario.",
+        "quick_zoom": "Acceder a Zoom",
+        "quick_teams": "Acceder a Teams",
+        "copy_passcode": "Copiar contraseña",
+        "self_study": " (Autoestudio) "
```
Comment (yrccondor) [inline src/components/calendar.vue:921]: no space is needed **in the end**

File: src/components/settings.vue [hunk 5]
```diff
@@ -244,6 +244,22 @@ export default {
...
+        "uom_username": "UoM nombre de usuario",
+        "uom_email": "UoM correo",
+        "uom_password": "UoM contraseña de cuenta",
+        "allow_save": "Usar este servidor back-end para sincronizar ajustes",
+        "will_store_on": "Habilitado permitirá el servidor back-end guardar la información de su cuenta UoM",
+        "not_allow_account": "Este servidor back-end no permite guardar la cuenta UoM",
+        "not_allow_email": "Este servidor back-end no permite la sincronización del correo",
+        "not_allow_sync": "Este servidor back-end no permite la sincronización de ajustes",
+        "where_ica": "Dónde puedo obtener mi subscripción del calendario",
+        "where_ica_body": "Vaya a <a href=\"https://timetables.manchester.ac.uk/\" target=\"_blank\" rel=\"noreferrer noopener\">, esta página</a>，y tras iniciar sesión haga clic en el botón de \"Subscribe\" en la parte de arriba derecha，más tarde a \"More\", \"Copy\"，respectivamente, y copie el URL obtenido.",
```
Comment (yrccondor) [inline src/components/settings.vue:259]: a comma within the a element and a full-width comma after the a element

**Review by yrccondor — CHANGES_REQUESTED** [review]

**Review by yrccondor — APPROVED** [review]
Cool, good job!

**Comment by yrccondor** [comment]
build faild again, re-check needed

**Review by yrccondor — APPROVED** [review]
great, we can merge now

**Pull request merged**